manifests/base/cluster-role.yaml

```diff
@@ -3,8 +3,8 @@ apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRole
 metadata:
   labels:
-    app: tf-job-operator
-  name: tf-job-operator
+    app: training-operator
+  name: training-operator
 rules:
 - apiGroups:
   - kubeflow.org
```

Comment:
We will need permissions to operator other jobs as well. Can you update resources? https://github.com/kubeflow/tf-operator/blob/c60dabb7daa3ff9ffb4eecfc24ee4a39bce4b7ca/config/rbac/role.yaml#L10-L18